pkgs/applications/networking/instant-messengers/qq/default.nix

```diff
@@ -19,7 +19,8 @@
 , vips
 , at-spi2-core
 , autoPatchelfHook
-, makeWrapper
+, wrapGAppsHook
+, commandLineArgs ? ""
```

Comment:
But why? Didn't you say we should use https://github.com/NixOS/nixpkgs/blob/04e15baaadf09ff9043458d84459d51d94ea2501/pkgs/applications/networking/instant-messengers/signal-desktop/generic.nix#L160-L164 instead?

Reply:
> But why? Didn't you say we should use
> 
> https://github.com/NixOS/nixpkgs/blob/04e15baaadf09ff9043458d84459d51d94ea2501/pkgs/applications/networking/instant-messengers/signal-desktop/generic.nix#L160-L164
> instead?

Because we would like to resolve two usages in one pr:
- Someone would like pass custom options (like https://github.com/NixOS/nixpkgs/pull/284788) so that `qq` can runs better on their machines.
- However, someone else (like me) simply wanna `qq` runs. I do not really care about command line options.

If we only use: https://github.com/NixOS/nixpkgs/blob/04e15baaadf09ff9043458d84459d51d94ea2501/pkgs/applications/networking/instant-messengers/signal-desktop/generic.nix#L160-L164
Then this pr does not helpful for the first usage.

I do not want to break someone else's usage, so that is why I instead open this pr.

Reply:
but we need to use `gappsWrapperArgs` instead of `makeWrapper`, right?

Reply:
> but we need to use `gappsWrapperArgs` instead of `makeWrapper`, right?

Exactly. That is why I also created a separate commit along this pr: https://github.com/NixOS/nixpkgs/commit/e6382fee6e41023e6cd02f75ffd0b09ac0f4255a .